Automattic/jetpack
# Update illustration colors to use Color Studio v2

Part of our Hack week project to update the illustrations to use the new [Color Studio v2](https://color-studio.blog/) that dropped last week. 

#### Changes proposed in this Pull Request:

* Updates existing illustrations with new colors.

**Old Illustrations (Free Plan)**
![L8xVpZA8Rg-3000x3000](https://user-images.githubusercontent.com/4297811/62090889-3a5b7480-b20a-11e9-866d-d4481fd54ea7.png)

**New Illustrations (Free Plan)**
![Screenshot_2019-07-29 Jetpack ‹ Hollow Macaroni — WordPress](https://user-images.githubusercontent.com/4297811/62090893-3deefb80-b20a-11e9-9c61-000d15ae4930.png)

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Not a new feature, just updating assets. More info here: p6TEKc-2SC-p2

#### Testing instructions:
* Activate and Connect Jetpack
* Go to My Plan
* Note that the illustrations have changed from drab to fab (tm)

#### Proposed changelog entry for your changes:

* Update illustration colors
